tensorboard/uploader/uploader_test.py

```diff
@@ -838,9 +779,11 @@ def test_remembers_first_metadata_in_scalar_time_series(self):
         tag_counts = {tag.name: len(tag.points) for tag in run_proto.tags}
         self.assertEqual(tag_counts, {"loss": 2})
 
-    def test_v1_summary_single_value(self):
+    def test_expands_multiple_values_in_event(self):
```

Comment:
This is one place where the diff gets confusing.

* test_v1_summary_single_value() has been migrated to ScalarBatchedRequestSenderTest as test_v1_summary().
* test_v1_summary_multiple_value() is kept here but renamed to test_expands_multiple_values_in_event(), which I think is its real value.